invenio_administration/views/base.py

```diff
@@ -300,16 +293,15 @@ def get(self, pid_value=None):
         schema = self.get_service_schema()
         serialized_schema = self._schema_to_json(schema)
         form_fields = self.form_fields
-        ui_schema = self.set_ui_schema_params(serialized_schema, self.form_fields)
-        # TODO context processor?
         return self.render(
             **{
-                "resource_schema": ui_schema,
+                "resource_schema": serialized_schema,
```

Comment:
I think this will fail in the UI as we do need the `ui_schema` to format the value of the field.
If I am not mistaken, by removing the `ui_schema` I think that we should also update the [Formatter](https://github.com/inveniosoftware/invenio-administration/blob/main/invenio_administration/assets/semantic-ui/js/invenio_administration/src/components/Formatter.js#L26) that was using this to set the correct `Element` to be used.

Reply:
we don't I have updated it in invenio-administration